Directory.Packages.props

```diff
@@ -0,0 +1,75 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
```

Comment:
could you separate libs from analyzers in two groups please?

Reply:
Updated (used SK as a reference).